starter-code/public/scripts/controllers/aboutController.js

```diff
@@ -7,9 +7,10 @@ var app = app || {};
   aboutController.index = () => {
     $('#about').show().siblings().hide(); // REVIEW: We have a slight refactor in selectors here, which has reduced the amount of code from the last lab.
 
-    // TODO: Call a function to load all the data.
+    // DONE: Call a function to load all the data.
     // Pass a view function as a callback, so the view will render after the data is loaded.
-
+    // estimated 15 min | actual 20 min
+    module.requestRepos(app.repoView.index);
```

Comment:
you should stick with using either app or module here. Also, requestRepos is attached to an object, so you will need to put that here before requestRepos but after module/app